Library/Formula/ffmpeg.rb

```diff
@@ -8,8 +8,8 @@ class Ffmpeg < Formula
     # fixed upstream, can be removed in the next release.
     # https://trac.ffmpeg.org/ticket/4238
     patch do
-      url "http://git.videolan.org/?p=ffmpeg.git;a=commitdiff_plain;h=83808ee7e9e89b0c83d5ecd94b35952c44716b68"
-      sha1 "918586c158e1231a8d59a78e6f05cedc83824d68"
+      url "https://git.videolan.org/?p=ffmpeg.git;a=commitdiff_plain;h=83808ee7e9e89b0c83d5ecd94b35952c44716b68"
```

Comment:
Maybe use this: https://github.com/FFmpeg/FFmpeg/commit/83808ee7e9e89b0c83d5ecd94b35952c44716b68.diff